dev/VSIX/ProjectTemplates/Desktop/CSharp/UnitTestApp/UnitTests.cs

```diff
@@ -5,23 +5,22 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace $safeprojectname$
+namespace $safeprojectname$;
+
+[TestClass]
+public class UnitTest1
```

Comment:
```suggestion
public partial class UnitTest1
```
Not needed for AOT in tests, but there's no overhead in doing this, will help folks just have/learn this, and there are more potential use of source generators for tests that have or are being built in this space, so future-proof us. ☺️

Reply:
`partial` added